### PR TITLE
Filter runs list for Groups

### PR DESCRIPTION
**Pull request by evantahler**
Fixes a bug when viewing the list of Runs created by a Group that was showing all the Runs

### Development

- [ ] Application changes have been tested appropriately

### Impact

- [ ] Code follows company security practices and guidelines
- [ ] Security impact of change has been considered
- [ ] Performance impact of change has been considered
- [ ] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

>

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [ ] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: ui/ui-components/components/runs/list.tsx [hunk 1]
```diff
@@ -41,6 +41,7 @@ export default function RunsList(props) {
   async function load() {
     const params = { limit, offset, topic };
     if (router.query.sourceId) params["id"] = router.query.sourceId.toString();
+    if (router.query.groupId) params["id"] = router.query.groupId.toString();
```
Comment (pedroslopez) [inline ui/ui-components/components/runs/list.tsx]: this `id` param is quite weird to me, when I did the source one I got confused. Don't have to change this now, but
I'm thinking maybe it should be more like `creatorId`

**Review by pedroslopez — APPROVED** [review]
This looks good, but it looks like the runs list is also included in a `property`'s page, should we do the same for them?

File: ui/ui-components/components/runs/list.tsx (hunk 1 above)
Comment (evantahler) [reply ui/ui-components/components/runs/list.tsx]: Good call! Updated via 816c9ea755da10460fc0638c502534574b58ae14

File: ui/ui-components/components/runs/list.tsx [hunk 2]
```diff
@@ -40,7 +40,8 @@ export default function RunsList(props) {
 
   async function load() {
     const params = { limit, offset, topic };
-    if (router.query.sourceId) params["id"] = router.query.sourceId.toString();
+    params["creatorId"] =
```
Comment (bleonard) [inline ui/ui-components/components/runs/list.tsx:43]: in general, it's a risk here by not including the `creatorType` of ids that match across these different models.

**Pull request merged**